test/http.test.js

```diff
@@ -160,34 +154,31 @@ test('http response support', async ({ ok, same, error, teardown }) => {
   server.close()
 })
 
-test('http response support via a serializer', async ({ ok, same, error, teardown }) => {
+test('http response support via a serializer', async ({ match, error }) => {
```

Comment:
can you please keep the previous test and only add a new one?

Reply:
Original tests restored. Also fixed linter issues.